src/bin/helper.rs

```diff
@@ -22,40 +31,58 @@ struct Args {
     scheme: Scheme,
 }
 
-fn main() {
-    unimplemented!();
+fn config() -> (NetworkConfig, ServerConfig) {
```

Comment:
Can we load this from a file instead?  Having a default value is good, but I get nervous seeing large r# strings.  (Deferring this is OK.)